mrtt-ui/src/views/Landscapes.js

```diff
@@ -90,9 +96,9 @@ function Landscapes() {
     <ContentWrapper>
       <TitleAndActionContainer>
         <PageTitle>{language.pages.landscapes.title}</PageTitle>
```

Comment:
This one worked, but with a warning about how component isnt a legal prop and something about refs

Reply:
so is there still a warning displaying? was it there before too? @mmnoo

Reply:
no warning anymore. sorry that was worded unclearly. This removed the warning.